src/carbonio-files-ui-common/views/components/AdvancedSearchModalContent.test.tsx

```
@@ -40,8 +40,7 @@ describe('Advanced search modal content', () => {
 				filters={filters}
 				closeAction={closeAction}
 				searchAdvancedFilters={searchAdvancedFilters}
-			/>,
-			{ mocks: {} }
```

The empty mocks object here was a way to declare (and check) that there are no network requests coming from this component. By removing it, the test will use default msw handlers, hiding the fact that possible network requests will be performed. I'm torn between leaving the empty object or not.